test/test_download.py

```diff
@@ -117,7 +117,7 @@ def print_skipping(reason):
 
         for other_ie in other_ies:
             if not other_ie.working():
-                print_skipping('test depends on %sIE, marked as not WORKING' % other_ie.ie_key())
+                print_skipping('test depends on %sIE, is _REPORTED_BROKEN' % other_ie.ie_key())
```

Comment:
If we touch it, should we
```suggestion
                print_skipping('test depends on {other_ie.ie_key()}IE, is _REPORTED_BROKEN')
```